packages/block/README.md

```diff
@@ -53,15 +55,31 @@ try {
 }
 ```
 
-# API
+## EIP-1559 Blocks
 
-[Documentation](./docs/README.md)
+This library supports the creation of [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559) compatible blocks starting with `v3.3.0`.
+
+To instantiate an EIP-1559 block the hardfork parameter on the `Common` instance needs to be explicitly set to `london` (default is still `istanbul`):
 
-# CONSENSUS TYPES
+```typescript
+import { Block } from 'ethereumjs-block'
+import Common from '@ethereumjs/common'
+const common = new Common({ chain: 'mainnet', hardfork: 'london' })
+const block = Block.fromBlockData({
+  header: {
+    //...,
+    baseFeePerGas: new BN(10),
+  }
+}, { common })
+```
+
+EIP-1559 blocks have an extra `baseFeePerGas` field (default: `new BN(7)`) and can encompass `FeeMarketEIP1559Transaction` txs (type `2`) (supported by `@ethereumjs/tx` `v3.2.0` or higher) as well as  `Transaction` legacy txs (internal type `0`) and `AccessListEIP2930Transaction` txs (type `1`).
+
+## Consensus Types
```

Comment:
This diff here looks a bit strange, better to be viewed as a complete file in rendered format.